src/components/App/App.test.js

```diff
@@ -55,7 +58,8 @@ describe('<App />', () => {
       user: {
         id: 123,
         name: 'Doe'
-      }
+      },
+      theme: { name: 'default' }
```

Comment:
If the component is mocked above, why is this prop needed?

Reply:
Removed.